semgrep/semgrep/formatter/sarif.py

```diff
@@ -55,6 +55,14 @@ def _rule_to_sarif(rule: Rule) -> Dict[str, Any]:
         if rule_url is not None:
             rule_json["helpUri"] = rule_url
 
+        rule_short_description = rule.metadata.get("shortDescription")
```

Comment:
We have plans in the next 3-6 months to overhaul the rule schema to include short and long descriptions at the top level as well as tags. For now I think it would be better to not rely on too much of this functionality in the optional meta field of the rule. Can we get away in the interim having the short description and help field just be the rule message as well?

Reply:
The rule message doesn't give enough information to our developers. We need to provide more context and having everything defined as the message is not sufficient. Can we use this as an interim solution? We can reconsider the approach once the rule schema overhaul is completed.

Reply:
Can you elaborate on what information is missing from a message and how this solves that need? Is there a reason context cannot be added to the message?

Reply:
Messaged out of band: 

> GitHub uses the shortDesription as the title for the alert and fullDescription as the subtitle for the alert. We want to keep the title short and the subtitle a bit longer explaining what exactly the alert is. We’re fine with using the fullDescription as the message but we’d also like to have a separate help text that guides the developers with helpful text and links which is also currently missing. As for the tags, we have tags like security, object-deserialization, external/cwe/cwe-502 etc. that is also not being captured.